src/main/java/org/jvnet/hudson/test/BuildWatcher.java

```diff
@@ -90,7 +89,7 @@
                 return;
             }
             RunningBuild build = new RunningBuild(r);
-            RunningBuild orig = builds.put(r.getLogFile(), build);
+            RunningBuild orig = builds.put(r.getRootDir(), build);
```

Comment:
We just needed a unique identifier of a build × `$JENKINS_HOME`; were not actually using the log file per se.